guides/guide.txt

```diff
@@ -0,0 +1,18 @@
+TOPIC
+    guide
+
+DESCRIPTION
+    A guide is a person who leads travelers, sportsmen, or tourists through
+    unknown or unfamiliar locations. The term can also be applied to a person who
+    leads others to more abstract goals such as knowledge or wisdom. 
```

Comment:
```suggestion
    A guide is a person who leads travelers through
    unknown or unfamiliar locations. The term can also be applied to a person who
    leads others to more abstract goals such as knowledge or wisdom. 
```
I think just leaving this at travelers is more effective (and I want to remove the gendered "sportsmen")